client/components/virtual-list/index.jsx

```diff
@@ -1,13 +1,24 @@
 import { AutoSizer, List } from '@automattic/react-virtualized';
 import classNames from 'classnames';
-import { localize } from 'i18n-calypso';
-import { debounce, range } from 'lodash';
+import { debounce } from 'lodash';
 import PropTypes from 'prop-types';
-import { cloneElement, Component } from 'react';
+import { cloneElement, createRef, Component } from 'react';
 
 const noop = () => {};
 
-export class VirtualList extends Component {
+function range( start, end ) {
+	if ( end < start ) {
+		return range( end, start ).reverse();
+	}
+	const length = end - start + 1;
+	const result = Array( length );
+	for ( let i = 0; i < length; i++ ) {
+		result[ i ] = start + i;
+	}
+	return result;
```

Comment:
Perhaps we could shorten this bit to something like:

```suggestion
	return Array.from( { length }, ( _, i ) => i + start );
```

Reply:
Good idea, implemented in db06b07c08 :+1: